Drivers/MotorChannel/Inc/ZP_D_PWMChannel.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef ZP_D_PWM_CHANNEL_HPP_
+#define ZP_D_PWM_CHANNEL_HPP_
+
+#include "main.h"
+
+#include "ZP_D_MotorChannel.hpp"
+
+class PWMChannel : public MotorChannel {
+    public:
+        PWMChannel(TIM_HandleTypeDef* timer,
+					uint16_t timer_channel);
+
+        void set(uint8_t percent);
+    private:
+    	//set to minimum and max amount of counts in a duty cycle
+        uint32_t MIN_SIGNAL;
+        uint32_t MAX_SIGNAL;
+        uint32_t PERIOD_TICKS_;  
+        TIM_HandleTypeDef *TIMER_;     //handle to the timer
```

Comment:
These are non `const`, so I'm pretty sure they shouldn't be all caps?
https://uwarg-docs.atlassian.net/wiki/spaces/ZP/pages/2268692486/Style+Standards#Member-Variables